src/Testing/src/xunit/SkipOnHelixAttribute.cs

```diff
@@ -65,7 +65,7 @@ private bool ShouldSkip()
         return Queues.ToLowerInvariant().Split(';').Contains(targetQueue);
     }
 
-    public static bool OnHelix() => !string.IsNullOrEmpty(GetTargetHelixQueue());
+    public static bool OnHelix() => HelixHelper.OnHelix();
```

Comment:
Suggest updating `SkipNonHelixAttribute` and `SkipOnCIAAttribute` too

Reply:
Good idea, done